Crtx 239728 ioa deprecation (#44334)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/2_11_0.md

#### Classifiers

> ##### CrowdStrike Falcon Incident Classifier

Reviewer: @talzich is this also relevant for XSIAM? If so, XSIAM uses issues, not incidents

> - Fixed an issue where Indicator of Attack detections were not classified to the correct incident type.

Reviewer: ```suggestion
- Fixed an issue where Indicator of Attack detections were not classified as the correct incident type.
```

#### Integrations

##### CrowdStrike Falcon

> - Fixed an issue where the Indicator of Attack fetch type would access a deprecated CrowdStrike service.

Reviewer 2: Better to mention that we are now using new endpoints and that the last one has been deprecated.

> - Updated the Indicator of Attack fetch type to use the new /alerts group of endpoints rather than the, now deprecated, /detects/entities/ioa/v1 endpoint.

Reviewer: ```suggestion
- Updated the Indicator of Attack fetch type to use the new /alerts group of endpoints instead of the deprecated /detects/entities/ioa/v1 endpoint.
```

> - ***Breaking changes***: Indicator of Attack (IOA) filter query now uses FQL.

Reviewer: ```suggestion
- **Breaking changes**: The *Indicator of Attack (IOA)* filter query now uses **FQL**.
```
